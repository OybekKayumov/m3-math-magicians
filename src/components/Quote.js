import './pages.css';

const Quote = () => {
  const quoteText = '"Your time is limited, so don’t waste it living someone else’s life. Don’t be trapped by dogma – which is living with the results of other people’s thinking."';
  const quoteAuthor = '- Steve Jobs';

  return (
    <div className="quote-container">
      <p>{quoteText}</p>
      <span>{quoteAuthor}</span>
    </div>
  );
};

export default Quote;
