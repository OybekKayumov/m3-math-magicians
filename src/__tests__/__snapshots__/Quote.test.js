import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../../components/Quote';

it('render correctly when there are no items', () => {
  const quote = renderer.create(<Quote />).toJSON();

  expect(quote).toMatchSnapshot();
});
