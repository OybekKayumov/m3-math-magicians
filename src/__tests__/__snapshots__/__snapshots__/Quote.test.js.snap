// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly when there are no items 1`] = `
<div
  className="container-quote"
>
  <p>
    "Your time is limited, so don’t waste it living someone else’s life. Don’t be trapped by dogma – which is living with the results of other people’s thinking."
  </p>
  <span>
    - Steve Jobs
  </span>
</div>
`;
